File: js/sumPairs.js
sumPairs = function(arr, num) {
    let output = [];
    let toSkip = [];
    for (element of arr) {
        let currentNum = element;
        toSkip.push(currentNum);
        for (subElement of arr) {
            if ((currentNum + subElement == num) && (!(toSkip.includes(subElement)))) {
                output.push([currentNum, subElement]);
            };
        };
    };
    if (output.length != 0) {
    return output;
    };
    return "unable to find pairs"
};

module.exports = {sumPairs};

File: python/sum_pairs.py
def sum_pairs(arr, num):
    output = []
    to_skip = ()
    for element in arr:
        current_num = element
        to_skip += (current_num,)
        for sub_element in arr:
            if sub_element + current_num == num and sub_element not in to_skip:
                sub_arr = [current_num, sub_element]
                output.append(sub_arr)
    if output == []:
        return "unable to find pairs"
    return output